ouroboros-network-framework/testlib/Ouroboros/Network/ConnectionManager/Test/Utils.hs

```diff
@@ -180,6 +184,9 @@ validTransitionMap t@Transition { fromState, toState } =
       (_                        , TerminatedSt)                     -> (33, show t)
       (_                        , UnknownConnectionSt)              -> (34, show t)
       (TerminatingSt            , UnnegotiatedSt Inbound)           -> (35, show t)
+      (UnnegotiatedSt Outbound      , InboundIdleSt Duplex)         -> (36, show t)
+      (UnnegotiatedSt Outbound      , InboundIdleSt Unidirectional) -> (37, show t)
+      (InboundIdleSt Unidirectional , OutboundUniSt)                -> (38, show t)
```

Comment:
The numbers on the right are duplicate, e.g. there are other transitions above with the same number.  Is this ok?

Reply:
ugh you are right, that's a bug. The numbers are needed for the coverage tests